MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -918,6 +918,30 @@ protected override string GetVideoArguments(StreamState state, EncodingOptions e
             return args;
         }
 
+        private NumberFormatInfo _ffmpegSegmentTimeDeltaNumberFormat = null;
+        protected NumberFormatInfo ffmpegSegmentTimeDeltaNumberFormat
+        {
+            get
+            {
+                if (_ffmpegSegmentTimeDeltaNumberFormat == null)
+                {
+                    NumberFormatInfo nfi = new NumberFormatInfo();
+                    nfi.CurrencySymbol = "";
+                    nfi.NaNSymbol = "0";
+                    nfi.NegativeInfinitySymbol = "0";
+                    nfi.NegativeSign = "";
+                    nfi.NumberDecimalDigits = 3;
+                    nfi.NumberDecimalSeparator = ".";
+                    nfi.NumberGroupSeparator = "";
+                    nfi.PositiveInfinitySymbol = "0";
+                    nfi.PositiveSign = "";
+                    nfi.PercentSymbol = "0";
+                    _ffmpegSegmentTimeDeltaNumberFormat = nfi;
+                }
+                return _ffmpegSegmentTimeDeltaNumberFormat;
+            }
+        }
+
```

Comment:
I don't really like this initialization and would prefer it either in constructor or in a static initializer like https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-constructors

Reply:
I looked at this and found out BaseStreamingService seems to be the grandfather of this class and moved it there, I also removed some NumberFormat things that seemed redundant.
According to https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo?view=netstandard-2.0#Instantiating_Invariant there exists an InvariantInfo type that I think would be better as long as we don't need to make further modifications.

Reply:
I pushed this in hawken93/jellyfin on branch ffmpeg-bugs-delta_time :)

Reply:
I don't think using the Invariant type is a good idea because of the group separators. The number should never ever reach >1000 however, so at that point we probably have bigger problems. Same goes with NaN, negative numbers, etc. That's why I added those in the first place. This way where you only disable the group separators is probably the best.

Reply:
That's fair enough, but blanking currency symbol (not going to be used, can be left to defaults) as well as blanking the negative sign seems incorrect. ffmpeg can understand negative numbers, but if the intention is to switch the sign, then it's best to do it mathematically instead of in the string formatting